content/docs/user-guide/experiment-management/experiments.md

```diff
@@ -0,0 +1,33 @@
+## Experiments
+
+_New in DVC 2.0_
+
+`dvc exp` commands let you automatically track a variation to an established
```

Comment:
not for this PR: `track a variation to an established data pipeline` sounds very complicated. 

`dvc exp run`, `dvc exp show`, and other `dvc exp` commands automatically capture and save experiment runs, including code, data, metrics, models, etc. 

or even better - get rid of this sentence :) It does more harm than good to my mind.